### DISCUSSION TITLE
[Feature] Different sort possibilities on the people ovderview page

### DISCUSSION THREAD

**Discussion by kingvuke** [immich-app/immich#5927] [Feature Request]
### The feature

Currently this page is sorted by number of photos in descending order.

Various sorting options, e.g. by name (ASC, DESC), last added, etc. would be great and helpful.

### Platform

- [x] Server
- [X] Web
- [ ] Mobile

**Comment by midiwidi** [comment]
I would like to add that a sorting option or filter for people not named yet would be very welcome. When naming people one has to scroll all the way down at the moment after each person named. That makes the process very slow. 
Sorting by name (wether ascending or descending) would probably already bring the people not named to the top so that no other sorting option might be required.

**Comment by bo0tzz** [comment]
#4879
